internal/controller/ols_app_postgres_reconciliator.go

```diff
@@ -38,33 +38,34 @@ func (r *OLSConfigReconciler) reconcilePostgresServer(ctx context.Context, olsco
 			Name: "reconcile Postgres PVC",
 			Task: r.reconcilePostgresPVC,
 		},
-		{
-			Name: "reconcile Postgres Deployment",
-			Task: r.reconcilePostgresDeployment,
-		},
+
 		{
 			Name: "generate Postgres Network Policy",
 			Task: r.reconcilePostgresNetworkPolicy,
 		},
+		{
+			Name: "reconcile Postgres Deployment",
+			Task: r.reconcilePostgresDeployment,
+		},
```

Comment:
@blublinsky Is there any significance to change reconcilePostgresDeployment after reconcilePostgresNetworkPolicy - I feel the original order is better.